features/post-with-keys.js

```diff
@@ -0,0 +1,19 @@
+var page = function () {
+    if (document.URL.startsWith("https://scratch.mit.edu/discuss")) {
+        return "f"; // F = forums
+    } else {
+        return "e"; // E = else (not forums)
+    }
+}
+
+(function() {
+    document.onkeydown = (e) => {
```

Comment:
For simplicity, let's use `document.addEventListener("keydown", function() {...})` instead.